website/screens/components/grid/code/GridCodePage.tsx

```diff
@@ -0,0 +1,327 @@
+          </td>
+          <td>
+            Sets the <Code>gap</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/gap"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            placeContent: 'normal' | 'start' | 'end' | 'center' | 'stretch' |
+            'space-between' | 'space-around' | 'space-evenly' | 'baseline' |
+            object
+          </td>
+          <td>
+            <Code>'normal'</Code>
+          </td>
+          <td>
+            Sets the <Code>place-content</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/place-content"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            placeItems: 'normal' | 'start' | 'end' | 'center' | 'stretch' |
+            'baseline' | object
+          </td>
+          <td>
+            <Code>'normal'</Code>
+          </td>
+          <td>
+            Sets the <Code>place-items</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/place-items"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>templateAreas: string[]</td>
+          <td></td>
+          <td>
+            Sets the <Code>grid-template-areas</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>templateColumns: string[]</td>
+          <td></td>
+          <td>
+            Sets the <Code>grid-template-columns</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>templateRows: string[]</td>
+          <td></td>
+          <td>
+            Sets the <Code>grid-template-rows</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-rows"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>as: keyof HTMLElementTagNameMap</td>
```

Comment:
Could you please change `as` type in Flex prop documentation to have this same type? `keyof HTMLElementTagNameMap`.
Now it has this type: 
![image](https://user-images.githubusercontent.com/10975289/235881875-806efe6f-6b3a-4771-af42-11cabbd4d195.png)

Reply:
Yes!